### PR TITLE
Remove ILL indirect deprecated code

### LINKED ISSUE #20083: Remove deprecated ILL Indirect code
LoadILLIndirect v1, IndirectILLReduction, ILLIN16BCalibration, and ILLCalibration UI tab deprecated since v3.9 can be now removed.

### PR DESCRIPTION
**Pull request by gvardany**
This PR removes the old version of the loader, 2 deprecated algorithms, and one GUI tab for indirect ILL.

**To test:**
1. Set at ISIS, go to `Interfaces>Indirect>Data Reduction`
2. You should see 7 tabs, from which first 3 specific to ISIS
3. Set at ILL, go to `Interfaces>Indirect>Data Reduction`
4. Now you should see 5 tabs, only first of which specific to ILL (Energy Transfer).
5. [Optional] If you are familiar with Indirect, check if there is something else that needs to be removed and I missed?

Fixes #20083 . 

**Release Notes** 

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####

- [x] Is the code of an acceptable quality?
- [x] Does the code conform to the [coding standards](http://www.mantidproject.org/Coding_Standards)?
- [ ] Are the unit tests small and test the class in isolation?
- [ ] If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- [x] Do changes function as described? Add comments below that describe the tests performed?
- [ ] Do the changes handle unexpected situations, e.g. bad input?
- [ ] Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: MantidQt/CustomInterfaces/CMakeLists.txt [hunk 1]
```diff
@@ -21,8 +21,7 @@ set ( SRC_FILES
 	src/Indirect/ConvFit.cpp
 	src/Indirect/CorrectionsTab.cpp
 	src/Indirect/DensityOfStates.cpp
-	src/Indirect/Elwin.cpp
-	src/Indirect/ILLCalibration.cpp
+        src/Indirect/Elwin.cpp
```
Comment (KarlPalmen) [inline MantidQt/CustomInterfaces/CMakeLists.txt]: indented?

**Comment by KarlPalmen** [comment]
You need to merge master into your branch and resolve the conflicts.

To ensure it, builds properly, you may need to follow the advice recently given by @martyngigg .

**Review by KarlPalmen — CHANGES_REQUESTED** [review]

**Comment by gvardany** [comment]
I have merged the master and solved the conflicts, can you please have another look?

**Comment by KarlPalmen** [comment]
Another conflict needs resolving in the ``ui.rst`` file.

**Review by KarlPalmen — APPROVED** [review]

**Pull request merged**